networkx/algorithms/centrality/tests/test_group.py

```diff
@@ -10,58 +10,97 @@
 class TestGroupBetweennessCentrality:
     def test_group_betweenness_single_node(self):
         """
-            Group betweenness centrality for single node group
+        Group betweenness centrality for single node group
         """
         G = nx.path_graph(5)
         C = [1]
-        b = nx.group_betweenness_centrality(G, C, weight=None, normalized=False)
-        b_answer = 3.0
+        b = nx.group_betweenness_centrality(
+            G, C, weight=None, normalized=False, endpoints=False
+        )
+        b_answer = [6.0]
         assert b == b_answer
 
     def test_group_betweenness_normalized(self):
         """
-            Group betweenness centrality for group with more than
-            1 node and normalized
+        Group betweenness centrality for group with more than
+        1 node and normalized
         """
         G = nx.path_graph(5)
         C = [1, 3]
-        b = nx.group_betweenness_centrality(G, C, weight=None, normalized=True)
-        b_answer = 1.0
+        b = nx.group_betweenness_centrality(
+            G, C, weight=None, normalized=True, endpoints=False
```

Comment:
Do you need to put `endpoints=False` on these tests? It is the default.
And can you why the answer is not the same value (2.0 instead of 1.0) as the old function?

Reply:
The reason is that in the previous implementation the shortest path from u to v in undirected graph is count as 1 (like betweenness.py). In my implementation (and in most of the papers that I know) the number of shortest path counts as 2. Should I change this to be like betweenness.py? should I write this in the docs?

Reply:
Looks like this is the last thing to discuss...

What do the references use? Is there a conflict in the literature? 
That would be indicated by any paper mentioning this discrepancy...

Reply:
The reference counts the number of shortest path in undirected graph from u to v as 2 (from u to v and from v to u). That is also the case in Brandes's article. I am not familiar with a paper stating that the number of shortest path between two nodes in undirected graph is 1, but I assume that there is one if betweenness.py did that. Should I state that in the description of the algorithm?

Reply:
Remember that if you use all starting nodes, you need to count each path as one because you will find the path from u to v and then again from v to u. So you count each undirected path twice even if you increment the counter by 1 for each direction.

Also, the mathematical formula given in the docstring sums over `s<t` so that indicates to me that you only count one path for v to u and u to v even though there are paths both directions.  This is an important difference (a factor of 2). So we certainly can't just change the algorithm so it returns a different value. What is the right value? Check the papers again for simple cases. I'll do that too.

Reply:
O.K. I see that I made two mistakes:
1) **undireced vs. directed path counting**: The definition of group betweenness centrality at the references is:
![image](https://user-images.githubusercontent.com/55619320/104605059-b491ea00-5686-11eb-9af4-209c6e2cc903.png)
This formula is true for both directed and undirected graphs.  So, we need to calculate the shortest path from s to t and from t to s. I change the formula in the docstring.
2) **normalize**: I made a mistake in the normalization. If the algorithm for directed and undirected graphs are the same, then the normalization factor should be the same.

Reply:
OK!   Good work finding those...
I think the answers now agree with the old version except for the case of unnormalized, undirected graphs.   The test `test_group_betweenness_single_node` shows that it is different (6.0 instead of 3.0). 

Can we just add at line 180 (just after the normalization):
```
        elif not G.is_directed():
            GBC_group /= 2
```
There might be a better way... Change the algorithm so you only do one direction of paths when G is undirected. That would presumably save computation time. But I'm fine just scaling the final answer for this PR. The speedup is significant alreay.

Reply:
The reason that `test_group_betweenness_single_node` shows different result is because G in this test is undirected. Do you want to change the result of the algorithm for the undirected unnormalized case, so it would agree with the old version (by dividing the result by 2)?

Reply:
Yes...  I'd like the answers to agree with the old version of the code -- just be faster. :)
That's the only case that differs now. Let's divide the result by 2 so it represents the number of undirected shortest paths through the group.